src/main/java/org/kairosdb/core/properties/Props.java

```diff
@@ -0,0 +1,10 @@
+package org.kairosdb.core.properties;
+
+public class Props {
+    public static final String ALIGN_DATAPOINT_TTL_WITH_TIMESTAMP =
+            "kairosdb.datastore.cassandra.align_datapoint_ttl_with_timestamp";
+    public static final String REPORTER_TTL = "kairosdb.reporter.ttl";
```

Comment:
Are these already defined elsewhere? Should let the primary site of usage define the name or consolidate all names in a single Props file? -- my gut says the latter does not align with the long-term config directionl.